devicon.json

```diff
@@ -5489,6 +5489,42 @@
         "color": "#fed500",
         "aliases": []
     },
+    {
+        "name": "laminas",
+        "altnames": [
+            "Laminas Project",
+            "Zend Framework"
+        ],
+        "tags": [
+            "php",
+            "framework",
+            "web",
+            "mvc",
+            "middleware"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "line",
+                "line-wordmark"
+            ],
+            "font": [
+                "original"
```

Comment:
Add, `original-wordmark`, `line`, and `line-wordmark` here as all 3 can be converted into font versions.